recruitment-service/.github/workflows/ci-cd-recruitment-service.yml

```
@@ -0,0 +1,77 @@
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Set up JDK 21
+      uses: actions/setup-java@v4
+      with:
+        java-version: '21'
+        distribution: 'temurin'
+        
+    - name: Set up application.properties
+      run : |
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)